README.md

```diff
@@ -59,7 +59,7 @@ Supported languages
 * **HLSL** ([*clang-format*](https://clang.llvm.org/docs/ClangFormat.html))
 * **HTML/XHTML/XML** ([*tidy*](http://www.html-tidy.org/))
 * **Hy** (Emacs)
-* **Java** ([*clang-format*](https://clang.llvm.org/docs/ClangFormat.html), [*astyle*](http://astyle.sourceforge.net/))
+* **Java** ([*clang-format*](https://clang.llvm.org/docs/ClangFormat.html), [*astyle*](http://astyle.sourceforge.net/)), [*google-java-format*](https://github.com/google/google-java-format)
```

Comment:
The closing parenthesis should be at the end of the line.

Reply:
That's funny I should enable smartparense for markdown mode